portal-web/docroot/html/taglib/ui/icon/link_content.jspf

```diff
@@ -42,11 +42,19 @@
 	</c:otherwise>
 </c:choose>
 
+<%
+String getLocalizeMessage = localizeMessage ? LanguageUtil.get(resourceBundle, message) : message;
```

Comment:
Since `LanguageUtil.get(...)` never returns `null`, the only way for this to act like that is if `message` is `null`, so we should simply test for that. You can see a similar approach in this same tag in https://github.com/liferay/liferay-portal/blob/master/portal-web/docroot/html/taglib/ui/icon/page.jsp#L57

Reply:
Okay, @jbalsas you are totally right the null value comes in the message and gets returned by https://github.com/liferay/liferay-portal/blob/76da930fa5d546b4b782e85737378267eb31c480/portal-impl/src/com/liferay/portal/language/LanguageImpl.java#L1009 

I'm trying now to identify the cause of this null, but I see it is coming from different places, we can't control the request attributes not to be null.

![image](https://user-images.githubusercontent.com/59687465/117315843-5cc2fd00-ae88-11eb-9523-39c7ab24d7a5.png)

I'm asking if it is a good solution to check if we receive a null message and just change it with an empty string here: https://github.com/liferay/liferay-portal/blob/76da930fa5d546b4b782e85737378267eb31c480/portal-web/docroot/html/taglib/ui/icon/init.jsp#L69

`String getMessage = (String)request.getAttribute("liferay-ui:icon:message");
String message = getMessage == null ? "" : getMessage;`

Regards.